okta-idx-kotlin/src/main/java/com/okta/idx/kotlin/client/InteractionCodeFlow.kt

```diff
@@ -41,10 +41,11 @@ import com.okta.idx.kotlin.dto.v1.Response as V1Response
 /**
  * The InteractionCodeFlow class is used to define and initiate an authentication workflow utilizing the Okta Identity Engine.
  */
-class InteractionCodeFlow constructor(private val client: OAuth2Client) {
+class InteractionCodeFlow(private val client: OAuth2Client) {
```

Comment:
I'm curious, is there a reason why the OAuth2Client is private? My Kotlin may be a bit rusty, but this will make it inaccessible to see by the developer, correct?

Reply:
That's correct. It's more of a legacy than an intentional decision to keep this private. I didn't think much about making this public because the developer can provide their own client to the constructor and have it available to them anyways. And it's also accessible to the developer through OAuth2Client.default if they construct the flow using the empty constructor. I can make this public